packages/create-cloudflare/e2e/tests/cli/cli.test.ts

```diff
@@ -634,6 +634,19 @@ describe("Create Cloudflare CLI", () => {
 			}
 		});
 	});
+
+	test("E2E: Incorrect platform specified for framework", async ({
```

Comment:
I am not sure what the best place for this test is, here it seems pretty appropriate to me, alternatively I could create a new test file in the [e2e/tests/frameworks](https://github.com/cloudflare/workers-sdk/tree/main/packages/create-cloudflare/e2e/tests/frameworks) directory, I am totally fine with either 🙂

Reply:
Well the idea is for the test to be more generic, I don't specifically care about the framework being solid, solid just to happen not to support Pages, maybe I can add a comment clarifying that?